nuxeo-liveconnect-box-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-liveconnect/nuxeo-liveconnect-box-link.html

```diff
@@ -45,6 +45,17 @@
         @apply --layout-center;
         @apply --layout-center-justified;
         @apply --nuxeo-liveconnect-anchor-layout;
+        color: var(--nuxeo-secondary-color, #0066ff);
+      }
+
+      a:hover {
+        color: var(--nuxeo-link-hover_-_color);
```

Comment:
Are these variables the ones generated/handled by the browser? 
In fact, we don't need to refer these kind of variables, since they are generated from our own variables (see [base theme](https://github.com/nuxeo/nuxeo-web-ui/blob/master/themes/base.html))

Reply:
PS: Not sure if we need to style `a:hover`